.devfile.yaml

```diff
@@ -64,17 +57,25 @@ commands:
         isDefault: true
   - id: buildimage
     exec:
-      label: "Build Image"
+      label: "4. Build Image"
       component: tools
       workingDir: ${PROJECT_SOURCE}
-      commandLine: "podman build -f src/main/docker/Dockerfile.jvm -t quay.io/che-incubator/inventory-quarkus ."
+      commandLine: |
+        IMAGE=image-registry.openshift-image-registry.svc:5000/openshift/quarkus-api-example
+        podman build -f src/main/docker/Dockerfile.jvm -t "${IMAGE}" .
       group:
         kind: build
-  - id: tag
+  - id: pushimage
     exec:
-      label: "Tag Image"
+      label: "5. Push Image"
       component: tools
-      workingDir: ${PROJECTS_ROOT}/inventory-quarkus
-      commandLine: "podman tag quay.io/coolstore-demo/inventory-quarkus:latest"
+      commandLine: |
+        # Requires `oc policy add-role-to-user registry-editor <user_name> -n openshift`
+        IMAGE=image-registry.openshift-image-registry.svc:5000/openshift/quarkus-api-example
```

Comment:
we don't use this, can we remove it? we use quay.io

Reply:
yes we can. The problem is that you need to login first then. this allow to push right away. if you then do a sed in the deployment yaml when applying it you can use this "inner loop" image.

Reply:
I may miss some context here but it's an important point so I will clarify.

When I am in my IDE I usually don't push to the prod registry (`quay.io/coolstore-demo`). Pipelines will push to prod registry after I git push / git commit. 

But when I am in my IDE I still want to push to a personal registry to test building the Dockerfile and running the application pod. For this purpose the internal OCP registry works fine: it's always there and doesn't require username / password.

Reply:
makes sense! thanks